custom/nic_compliance/utils.py

```diff
@@ -90,11 +95,15 @@ def record_login_attempt():
         client.set(key_name, obfuscated_passwords + [hash_password(obfuscated_password)])
         client.expire(key_name, timedelta(EXPIRE_LOGIN_ATTEMPTS_IN))
 
+    def _mobile_request_to_track():
+        return resolve(request.path).url_name in MOBILE_REQUESTS_TO_TRACK_FOR_REPLAY_ATTACK
```

Comment:
To be able to test this without effecting resources at scale, i would like to test this by adding a check for username like
`username == 'test-nic-compliance'`?
Does that sound okay or is there a better way?
I can't use FF or domain specific check here.

Reply:
Fine to hardcode a username for a test.